android/source/VideoLocker/src/org/edx/mobile/view/SplashActivity.java

```diff
@@ -26,9 +25,6 @@ protected void onCreate(Bundle savedInstanceState) {
         setContentView(R.layout.activity_splash);
         overridePendingTransition(R.anim.slide_in_from_right, R.anim.slide_out_to_left);
 
-        // initialize log util
-        LogUtil.init(this);
```

Comment:
@shahidtamboli We need to initialize logging module from <code>MainApplication</code>. Without this, logs will also be printed in release mode.